activation/atxdb.go

```diff
@@ -491,11 +495,32 @@ func (db *ActivationDb) IsIdentityActive(edId string, layer types.LayerID) (bool
 	if len(ids) == 0 { // GetIdentity succeeded but no ATXs, this is a fatal error
 		return false, fmt.Errorf("no active IDs for known node")
 	}
+
 	atx, err := db.GetAtx(ids[len(ids)-1])
 	if err != nil {
 		db.log.Error("IsIdentityActive erred while getting atx err=%v", err)
 		return false, nil
 	}
+
+	lastAtxTargetEpoch := atx.PubLayerIdx.GetEpoch(uint16(db.LayersPerEpoch)) + 1
```

Comment:
Add comment to explain what we are doing in this flow.
Also, add a note why it is enough to check only last two.